src/configuration.cpp

```diff
@@ -408,7 +400,7 @@ chimera::CompiledConfiguration::CompiledConfiguration(
         = [](const std::string &str) -> std::string { return str; };
 }
 
-chimera::CompiledConfiguration::~CompiledConfiguration()
+chimera::CompiledConfiguration::~CompiledConfiguration() noexcept(false)
```

Comment:
It is a bit weird to be doing this in the destructor.  I wonder if there is a different location in the visitor call chain where this would make more sense :thinking: